package.json

```diff
@@ -35,7 +35,7 @@
   },
   "devDependencies": {
     "@types/node": "^20.1.4",
-    "@wonderlandengine/api": ">=1.1.3 <1.3",
```

Comment:
use version ranges here. we don't know if 1.5 will be supported, just like how 1.4 wasn't supported here due to arrays being added